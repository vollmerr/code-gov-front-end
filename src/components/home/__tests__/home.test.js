import React from 'react';

import Home from 'components/home';

import { shallow } from 'enzyme';

describe('Testing <Home /> component', () => {
  let wrapper;
  beforeEach(() => {
    global.PUBLIC_PATH = '/home'
    wrapper = shallow(<Home />);
  });
  it('Testing if component renders', () => {
    expect(wrapper.exists()).toEqual(true);
  });
  it('should render correct', () => {
    expect(wrapper).toMatchSnapshot();
  })  
  it('componentDidMount', () => {
    const spy = jest.spyOn(Home.prototype, 'componentDidMount');
    const wrapper = shallow(<Home />);
    wrapper.instance().componentDidMount()
    expect(spy).toHaveBeenCalled();
  })
})