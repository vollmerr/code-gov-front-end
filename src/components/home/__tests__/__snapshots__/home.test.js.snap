// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Home /> component should render correct 1`] = `
<Fragment>
  <Connect(HomeBanner) />
  <Connect(HomeAbout) />
  <Connect(HomeFeaturedProjects) />
  <Connect(HomeProjectBoardComponent) />
  <HomePartners />
  <HomeConnect />
</Fragment>
`;
